usr/libexec/greenboot/greenboot.sh

```diff
@@ -2,75 +2,56 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-run_required_scripts () {
-  echo "Running Required Health Check Scripts..."
-  local required_scripts=`find /etc/greenboot/check/required.d -name '*.sh'`
+script_runner () {
+  if [ $# -eq 0 ]; then
```

Comment:
It's a bit odd to have this usage checking inside an implementation function. Let's maybe put it just before the main `case`?

Reply:
I'll put this into the "run" case